site/en/blog/page-lifecycle-api/index.md

```diff
@@ -76,26 +82,30 @@ state](#developer-recommendations-for-each-state) for the exceptions).
 Perhaps the easiest way to explain the Page Lifecycle states &mdash; as well as
 the events that signal transitions between them &mdash; is with a diagram:
 
-
-  <figure>
-<a href="https://wd.imgix.net/image/kheDArv5csY6rvQUJDbWRscckLr1/Hs3H9gK98YT0pvvU3k25.png">
-    {% Img src="image/kheDArv5csY6rvQUJDbWRscckLr1/Hs3H9gK98YT0pvvU3k25.png", alt="Page Lifecycle API state and event flow.", width="800", height="541" %}
-    </a>
-  </figure>
+<figure>
+  <a href="https://wd.imgix.net/image/eqprBhZUGfb8WYnumQ9ljAxRrA72/1mN2saGWo57O6vIOdcGy.svg">
+    {% Img
+      src="image/eqprBhZUGfb8WYnumQ9ljAxRrA72/KCIeOsJ0lCWMthBSSBrn.svg",
+      alt="Page Lifecycle API state and event flow.",
+      width="800",
+      height="400"
+    %}
+  </a>
+</figure>
 
 ### States
 
 The following table explains each state in detail. It also lists the possible
 states that can come before and after as well as the events developers can
 use to observe changes.
 
-<table class="cyan">
+<table>
   <tr>
     <th>State</th>
     <th>Description</th>
   </tr>
   <tr>
-    <td><code id="state-active">Active</code></strong></td>
+    <td><strong id="state-active">Active</strong></td>
     <td>
       <p>A page is in the <i>active</i> state if it is visible and has
```

Comment:
Totally minor accessibility nit, but my understanding is that `<i>` or `<b>` elements don't convey the same semantics as `<em>` and `<strong>`. You can adjust these if necessary, but not a big deal if you do not.

Reply:
I'm not an expert on this, but I was using `<i>` to denote that these are technical terms, as suggested here: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/i